docs/examples/example_lime.ipynb

```diff
@@ -359,7 +360,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "For guidelines on how to interpret such lime plots please see the [lime documentation](https://github.com/marcotcr/lime)."
+    "In these plots, the green bars signify that the corresponding feature referenced on the\n",
+    "y-axis, increases the CATE estimate. On the other side, the red bars, represent\n",
+    "features that reduce CATE estimates.\n",
```

Comment:
Does "increase" and "decrease" mean that if these features are added resp. removed from the model, then the CATE increases / decreases?

Reply:
Not exactly, it means that in the specific case of the explained instance and model, the presence of this variable makes the prediction increase/decrease, but this does not mean that removing it would decrease/increase.

Taken from a LIME tutorial: "the way to interpret this is: if odor was not foul, on average, this prediction would be 0.24 less 'poisonous'." 

Basically it means that if that changing that variable value, the final prediction would change on average by the value showed in the plot.
If you think further explanation is needed lmk!

Reply:
I see. Maybe "the green bars signify that the presence of a feature (y-axis) in the model increases the CATE estimate whereas red bars signify that a feature's presence in the model reduces the CATE."

Reply:
[ae34bed](https://github.com/Quantco/metalearners/pull/21/commits/ae34bed4c0ec3592472ecfe925bb9e373bab7186)